src/Components/modal/editModal.js

```diff
@@ -26,16 +31,13 @@ export const EdiPostModal = ({ isOpen, onClose }) => {
           <ModalHeader>Edit Tweet</ModalHeader>
           <ModalCloseButton />
           <ModalBody>
-            <Input onChange={(e) => setEditPost(e.target.value)} />
+            <Input onChange={(e) => setEdittedPost(e.target.value)} />
```

Comment:
When we click on edit , the input is not showing the post its blank , I think you missed to add value attribute which should hold the post we are editing.